packaging/conda/meta.yaml

```diff
@@ -61,13 +63,13 @@ outputs:
       - bin/khiops_coclustering.cmd # [win]
     test:
       commands:
-        - khiops_env --env 
+        - khiops_env --env
         - khiops -s
         - khiops_coclustering -s
 
   # kni package (do not need khiops-core as a runtime dependency)
   - name: kni
-    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    version: {{ version.replace("-", ".") }}
```

Comment:
I would substitute "-" for "" to be consistent with current versioning scheme (see comment above).